packages/react/src/components/card/Card.module.scss

```diff
@@ -11,6 +11,10 @@
   padding: var(--padding-vertical) var(--padding-horizontal);
 }
 
+.boxShadow {
```

Comment:
Why the box shadow is not implemented for Dialog component?

Reply:
The Dialog is always required to have an overlay behind it so the shadow won't do much.

But I agree, it probably should be allowed to the Dialog as well.

Reply:
- [ ] Add box shadow to Dialog component